src/_pytest/nodes.py

```diff
@@ -364,6 +364,14 @@ def _repr_failure_py(
     def repr_failure(
         self, excinfo, style=None
     ) -> Union[str, ReprExceptionInfo, ExceptionChainRepr, FixtureLookupErrorRepr]:
+        """
+        Return a string representation of a collection failure.
+
+        :param excinfo: Exception information for the failure as a tuple
+            (type, value, traceback), see sys.exc_info().
+        :param style: Style of the representation ('long', 'short', 'auto',
```

Comment:
I think this parameter is a bit suspicious and I'm not really it's supposed to be used directly.

Do you use it? If not, I'll err on the side of conservatism and not document it, for now.

Reply:
I don't use the `style` parameter in my override function. However, it was this very parameter that I first did not have in my override function and that was reported by Pylint. I'm fine with not documenting it in the docstring.